Update developer and user guide (#70)
AY2425S1-CS2113-W14-1/tp: docs/UserGuide.md

#### Adding a spending:

Adds an entry into user spending list. Entry will be displayed at the latest index. <br>
Run the [`list spendings`](#listing-all-spendings) command to display the list with the new entry. <br>
Date is optional and can be added to the end of the input, default would be set to current date. A date is enclosed in slashes. <br>
Tag is optional and can be added to the end of the input. A Tag is enclosed in asterisks. <br>

> 

Reviewer: ```suggestion
Recurrence Frequency is optional and can be added to the end of the input. A Recurrence Frequency is enclosed in tilde. <br>
```

#### Adding an income:

Adds an entry into user income list. Entry will be displayed at the latest index. <br>
Run the [`list incomes`](#listing-all-incomes) command to display the list with the new entry. <br>
Date is optional and can be added to the end of the input, default would be set to current date. A date is enclosed in slashes. <br>
Tag is optional and can be added to the end of the input. A Tag is enclosed in asterisks. <br>

> 

Reviewer: ```suggestion
Recurrence Frequency is optional and can be added to the end of the input. A Recurrence Frequency is enclosed in tilde. <br>

```

    ____________________________________________________________
```

### Exiting the program

The `bye` command allows you to exit the program safely, as it will store all changes made.

**Format**: `bye`

**Example input**:<br>
`bye`

**Example output**:
```
	____________________________________________________________
	Bye. Hope to see you again soon!
	____________________________________________________________
```

## FAQ

**Q**: How do I transfer my data to another computer? 

> **A**: Simply transfer `incomes.txt`, `spendings.txt` and `password.txt` files to the folder that the program is at.

Reviewer: ```suggestion
**A**: Simply transfer `incomes.txt`, `spendings.txt` and `password.txt` files to the directory containing the jar file.
```
